src/logic/embedding.lean

```diff
@@ -24,6 +24,8 @@ infixr ` ↪ `:25 := embedding
 
 instance {α : Sort u} {β : Sort v} : has_coe_to_fun (α ↪ β) := ⟨_, embedding.to_fun⟩
 
+initialize_simps_projections embedding (to_fun → apply)
```

Comment:
Once this PR goes in, it would be great to do this for the massive pile of `_hom` classes...